src/encoded/docs/schema-changes.md

```diff
@@ -154,44 +154,114 @@ This new object is an array of example objects that can successfully POST agains
+
+        "purpose":{
+            "title": "Purpose",
+            "description": "The purpose of the genetic modification.",
+            "type": "string",
+            "enum": [
+                "activation",
+                "analysis",
+                "overexpression",
+                "repression",
+                "tagging",
+                "validation",
+                "screening",
+                "expression"
+            ]
+        },
+        # Changing validation to characterization as a list of enums within the purpose property:
+        "purpose":{
+            "title": "Purpose",
+            "description": "The purpose of the genetic modification.",
+            "type": "string",
+            "enum": [
+                "activation",
+                "analysis",
+                "overexpression",
+                "repression",
+                "tagging",
+                "characterization",
+                "screening",
+                "expression"
+            ]
+        },
+
+2. In the **schemas** directory, edit the existing properties in the corresponding JSON file named after the object and increment the schema version. For example if the original schema version for the genetic modification object being modified was "6", change it to "7" (6->7): 
         
         "schema_version": {
-            "default": "2"
+            "default": "7"
         }
 
-2. In the **upgrade** directory add an ```upgrade_step``` to an existing/new python file named after the object. An example upgrades the object from schema version (2->3):
+2. In the **upgrade** directory add an ```upgrade_step``` to an existing/new python file named after the object. An example to the upgrade step is shown below. Continuing with our example, after the upgrade all the existing objects with that had "purpose" specified to be "validation" must now be changed to "characterization". And since the schema is changing from version 6 to 7 the def must specify this (6->7):
+
+        @upgrade_step('genetic_modification', '6', '7')
+        def genetic_modification_6_7(value, system):
+            if value['purpose'] == 'validation':
+                value['purpose'] = 'characterization'
 
-        @upgrade_step('{metadata_object}', '2, '3')
-        def {metadata_object}_2_3(value, system):
-            # {Redmine Issue}
+3. In the **tests/data/inserts** directory, we will need to change all the corresponding objects to follow the new schema. Continuing with our example, all the ```"purpose": "validation"``` must now be converted to ```"purpose": "characterization"```. So, we need to change all the corresponding inserts within the genetic modification object. For example:
```

Comment:
change "we will need to change" to "change all..." lie you did in the previous steps.